modules/survey_accounts/test/survey_accountsTest.php

```diff
@@ -246,6 +254,12 @@ function testSurveyAccountsAddSurvey()
 
         // Ensure visit label exists for a candidate.
         $this->safeGet($this->url . "/survey_accounts/");
+        $this->webDriver->wait(10, 1000)->until(
```

Comment:
Without this code the ajax request has not completed and the assertion fails (since the upgrade).